test/test_svd.cc

```diff
@@ -57,6 +58,26 @@ void test_svd_work( Params& params, bool run )
     params.ortho_V();
     params.error.name( "S - Sref" );
     params.error2.name( "Backward" );
+    if (timer_level >= 2) {
+        params.time2();
+        params.time3();
+        params.time4();
+        params.time5();
+        params.time6();
+        params.time7();
+        params.time8();
+        params.time9();
+        params.time10();
+        params.time2.name( "geqrf (s)" );
+        params.time3.name( "gelqf (s)" );
+        params.time4.name( "ge2tb (s)" );
+        params.time5.name( "tb2db (s)" );
+        params.time6.name( "bdsqr (s)" );
+        params.time7.name( "unmtr_hb2st (s)" );
+        params.time8.name( "unmbr_ge2tb (s)" );
+        params.time9.name( "unmqr (s) " );
+        params.time10.name( "unmlq (s)" );
```

Comment:
Rename and add 2 more timers, so we have these 6 timers, in this order:
7. `unmbr_tb2st_U (s)`
8. `unmbr_ge2tb_U (s)`
9. `unmqr (s)`  // which updates U
10. `unmbr_tb2st_V (s)`
11. `unmbr_ge2tb_V (s)`
12. `unmlq (s)`  // which updates V

Reply:
Or if we merge unmqr and unmlq, they can just be at the end:
7. `unmbr_tb2st_U`
8. `unmbr_ge2tb_U`
9. `unmbr_tb2st_V`
10. `unmbr_ge2tb_V`
11. `unmqr` or `unmlq`